hera_cal/tests/test_smooth_cal.py

```diff
@@ -144,6 +180,7 @@ def test_time_freq_2D_filter(self):
         np.testing.assert_array_almost_equal(ff, np.ones((100, 100), dtype=complex))
         ff, info = smooth_cal.time_freq_2D_filter(gains, wgts, freqs, times, filter_mode='plus')
         np.testing.assert_array_almost_equal(ff, np.ones((100, 100), dtype=complex))
+        ff, info = smooth_cal.time_freq_2D_filter(gains, wgts, freqs, times, method='DPSS')
```

Comment:
Can you add another `np.testing.assert_array_almost_equal(ff, np.ones((100, 100), dtype=complex))` after DPSS?

Reply:
Added this